Assignment 5.Rmd

```diff
@@ -75,3 +78,9 @@ D2$prediction <- predict(score_ctree, D2)
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
+
+```{r}
+mean(ifelse(D2$prediction == "no action", 1, 0))
+
+```
+A: Because compared to D1, all students achieved a score of 1 in D2, which means that no action should be carried out to help them, and thus the percentage of "no action" should be zero. As calculated above, "no action" takes up 58 percent of all the predictions, which is far more than zero percent for the actural advice.
```

Comment:
Great work! What do you think this means for the generalizability of your model?